### ISSUE TITLE
Admin console not displaying correctly

### ISSUE THREAD

**Issue by th4t9uy** [pi-hole/pi-hole#226]
Set up PiHole on a fresh Raspbian Jessie LITE install on a RaspberryPi Model B. 

However, not long after set up the admin console has started looking like this http://i.imgur.com/1RhcVuC.png

I have tested it in Firefox and Internet Explorer, I get the same result in either browser. On the RasPi so far all I have tried is restarting the lighttpd service.

**Comment by dschaper** [comment]
Okay, thanks for the report, we'll take a look at it...

**Comment by dschaper** [comment]
Do you have anything for your blocking page, or did you leave it with the stock blank html? And can you grab a page source? Thanks!

**Comment by PromoFaux** [comment]
Do you have any extensions installed? Ad block plus/Privacy Badger etc? If so.. can you try disabling them and refreshing? Thanks

**Comment by jacobsalmela** [comment]
I haven't seen this one yet, but try another browser and/or disabling the extensions.

**Comment by th4t9uy** [comment]
dschaper: I left it with the stock html, the only thing I changed during the initial of Pi-Hole setup was settting at a static IP address and enabled IPv6 blocking. Page source is as follows: http://pastebin.com/9n0SHn3Q

PromoFaux: I did initially have UBlock Origin and Disconnect Firefox addons enabled. Both have since been disabled but it didn't help unfortunately.

jacobsalmela: I have tried Firefox and Internet Explorer, the latter I doesn't have any extensions/addons running. I have also since tried Chrome from my phone but again that does not display correctly.

**Comment by dschaper** [comment]
Thanks for the update. Can you do an `ls -lac /var/www/html/admin' and `ls -lac /var/www/html/admin/css`

**Comment by th4t9uy** [comment]
Results of commands run:

    pi@raspberrypi:~ $ ls -lac /var/www/html/admin
    total 48
    drwxr-xr-x 5 root     root      4096 Jan 22 22:06 .
    drwxrwxr-x 4 www-data www-data  4096 Jan 22 22:06 ..
    -rw-r--r-- 1 root     root        62 Jan 22 22:06 api.php
    drwxr-xr-x 2 root     root      4096 Jan 22 22:06 css
    -rw-r--r-- 1 root     root        90 Jan 22 22:06 .gitignore
    drwxr-xr-x 2 root     root      4096 Jan 22 22:06 img
    -rw-r--r-- 1 root     root     11707 Jan 22 22:06 index.php
    drwxr-xr-x 2 root     root      4096 Jan 22 22:06 js
    -rw-r--r-- 1 root     root      1085 Jan 22 22:06 LICENSE
    -rw-r--r-- 1 root     root      1195 Jan 22 22:06 README.md
    pi@raspberrypi:~ $ ls -lac /var/www/html/admin/css/
    total 96
    drwxr-xr-x 2 root root  4096 Jan 22 22:06 .
    drwxr-xr-x 5 root root  4096 Jan 22 22:06 ..
    -rw-r--r-- 1 root root 81982 Jan 22 22:06 AdminLTE.min.css
    -rw-r--r-- 1 root root  3094 Jan 22 22:06 skin-blue.min.css
    pi@raspberrypi:~ $

**Comment by dschaper** [comment]
Those look good. How about a `cat /etc/lighttpd/lighttpd.conf`please?

**Comment by th4t9uy** [comment]
As follows: 

    pi@raspberrypi:~ $ cat /etc/lighttpd/lighttpd.conf
    server.modules = (
            "mod_access",
            "mod_accesslog",
            "mod_expire",
            "mod_compress",
            "mod_redirect",
            "mod_setenv",
            "mod_rewrite"
    )
    
    server.document-root        = "/var/www/html"
    server.error-handler-404        = "pihole/index.html"
    server.upload-dirs          = ( "/var/cache/lighttpd/uploads" )
    server.errorlog             = "/var/log/lighttpd/error.log"
    server.pid-file             = "/var/run/lighttpd.pid"
    server.username             = "www-data"
    server.groupname            = "www-data"
    server.port                 = 80
    accesslog.filename                      = "/var/log/lighttpd/access.log"
    accesslog.format                        = "%{%s}t|%V|%r|%s|%b"


    index-file.names            = ( "index.php", "index.html", "index.lighttpd.html" )
    url.access-deny             = ( "~", ".inc" )
    static-file.exclude-extensions = ( ".php", ".pl", ".fcgi" )
    
    compress.cache-dir          = "/var/cache/lighttpd/compress/"
    compress.filetype           = ( "application/javascript", "text/css", "text/html", "text/plain" )
    
    # default listening port for IPv6 falls back to the IPv4 port
    include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
    include_shell "/usr/share/lighttpd/create-mime.assign.pl"
    include_shell "/usr/share/lighttpd/include-conf-enabled.pl"
    
    # If the URL starts with /admin, it is the Web interface
    $HTTP["url"] =~ "^/admin/" {
            # Create a response header for debugging using curl -I
            setenv.add-response-header = ( "X-Pi-hole" => "The Pi-hole Web interface is working!" )
    }
    
    # If the URL does not start with /admin, then it is a query for an ad domain
    $HTTP["url"] =~ "^(?!/admin)/.*" {
            # Create a response header for debugging using curl -I
            setenv.add-response-header = ( "X-Pi-hole" => "A black hole for Internet advertisements." )
    }
    pi@raspberrypi:~ $

**Comment by dschaper** [comment]
That all looks good. Since you haven't made any changes to the configuration, would you re-run the `curl -L install.pi-hole.net | bash` command again and try a new install. This should just install over your existing configuration.

**Comment by th4t9uy** [comment]
All back to normal! http://i.imgur.com/XTM0QBL.png

Thanks for all your help guys

**Comment by dschaper** [comment]
Great! Thanks for using the Pi-Hole!

**Comment by ecaron** [comment]
I encountered this tonight too. I found the problem was a bunch of 0 byte files in /var/cache/lighttpd/compress/admin. I just rm -rf'd the directory and restarted lighttpd. Problem has not come back, yet.

I'll keep an eye on it happening and update this thread if I find more. But thought this might be useful for people before proceeding to perform a new install.

**Comment by dschaper** [comment]
Thanks for the report. Let us know if it does come back. (How'd you even think to look at the cache!)

**Comment by ecaron** [comment]
> How'd you even think to look at the cache!

Just following the rabbit hole. I was disabling parts of the /etc/lighttpd/lighttpd.conf file and things started working when I commented out the compression lines. Once I nuked the directory and removed the comments, some files started reappearing in the cache directory but not as many as before. The hunt continues! (I'm also on a Pi B, so maybe it has to do with not enough processing power to run the minifiers?)

**Comment by dschaper** [comment]
Well I have it running on a Zero, with a ton of empty overhead to grow into. It does take about 5 seconds or so to start rendering and 20 seconds until complete graph rendering and no more spinners.  I wouldn't think the 300MHz boost that the Zero has over the B+ would make that much impact?

**Comment by ecaron** [comment]
Excellent point. I apologize to the Pi for thinking it unfit:-) I'll keep
figuring out how to investigate the cause.

**Comment by dschaper** [comment]
:+1: I sent my 256MB models to greener pastures... If you do come up with a cache bug, open up a new Issue thread for it and we'll see if it's something that is upstream or a config issue.

**Comment by luebbe** [comment]
Hi Folks,
I had the same issue today. A fresh install of pi-hole on a vanilla raspbian, Raspberry B.
Everything worked fine until after an hour or so, the admin page looked almost like in the original posters screenshot.
I checked the config files, went through the re-install, deleted the cache, but nothing helped.
Only when I commented out "mod_compress" in the server.modules section of lighttpd.conf and restarted lighttpd, everything worked as expected immediately.
